job-runner.toml

```diff
@@ -47,12 +47,20 @@ dir = '~/.cache/cockpit-dev/job-runner-logs'
 link = 'http://127.0.0.1:8000/'
 
 [forge]
-driver='github'
+default = 'github'
 
 [forge.github]
+driver = 'github'
```

Comment:
I *think* with the merging we can leave https://github.com/cockpit-project/cockpituous/blob/main/ansible/roles/tasks-systemd/tasks/main.yml#L84 as-is, right?